src/components/relayPiifields.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import {graphql, useStaticQuery} from 'gatsby';
+
+const query = graphql`
+  query PiiFieldsQuery {
+    allPiiFieldPath {
+      nodes {
+        path
+      }
+    }
+  }
+`;
+
+function PiiField({field}) {
+  const asterisk_flag = field.endsWith('other');
+
+  return (
+    <React.Fragment>
```

Comment:
The `<React.Fragment>` is unnecessary here, since there is only one child (`<dt>`).